tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -724,16 +740,22 @@ async function waitForSubscriptionsTillReady(targetNamespace) {
   await waitForSubscription('order-created', targetNamespace);
 }
 
-async function checkInClusterEventDelivery(targetNamespace) {
-  await checkInClusterEventDeliveryHelper(targetNamespace, 'structured');
-  await checkInClusterEventDeliveryHelper(targetNamespace, 'binary');
-  await checkInClusterLegacyEvent(targetNamespace);
+async function checkInClusterEventDelivery(targetNamespace, testV1Alpha2CRD=false) {
+  await checkInClusterEventDeliveryHelper(targetNamespace, 'structured', testV1Alpha2CRD);
+  await checkInClusterEventDeliveryHelper(targetNamespace, 'binary', testV1Alpha2CRD);
+  await checkInClusterLegacyEvent(targetNamespace, testV1Alpha2CRD);
 }
 
 // send event using function query parameter send=true
-async function sendInClusterEventWithRetry(mockHost, eventId, encoding, retriesLeft = 10) {
+async function sendInClusterEventWithRetry(mockHost, eventId, encoding, eventType='', retriesLeft = 10) {
+  const eventData = {id: eventId};
+  if (eventType) {
+    eventData.save = true;
```

Comment:
What does `eventData.save = true` mean?

Reply:
The last-order function does not persist all received events, it was hard coded to only save events of type `order.received`. I changed it so that if a event data contains `save: true`, the last-order function will save this event so that later we can fetch and confirm if the event was received.